### DISCUSSION TITLE
[crash on startup] X Error BadDrawable

### DISCUSSION THREAD

**Discussion by SumianVoice** [bjornbytes/lovr#792] [Q&A]
Command line error:

```
X Error of failed request:  BadDrawable (invalid Pixmap or Window parameter)
  Major opcode of failed request:  149 ()
  Minor opcode of failed request:  4
  Resource id in failed request:  0x740000c
  Serial number of failed request:  186
  Current serial number in output stream:  193
```

System:

```
Linux Mint 20.1 Cinnamon, 4.8.6
Kernel 5.15.0-107-generic
CPU: AMD Ryzen 7 4800H with Radeon Graphics × 8
GPU: NVIDIA Corporation TU116M [GeForce GTX 1660 Ti Mobile]
```

I've completely ruined my build environment and can't even build anything from source whatsoever on my system and I don't know if that's related. Am I just doomed?

**Comment by bjornbytes** [comment]
Hm, you might be doomed.  Which version of the nvidia drivers do you have installed?  It seems like a lot of people on the internet have had similar errors, so I don't think it's a LÖVR specific issue.

**Reply by bjornbytes** [reply]
I think this is due to hybrid GPU (see #799)
